src/routes/issues/+page.svelte

```diff
@@ -45,37 +30,48 @@
 />
 
 <div style="display: flex;">
+	<!-- TODO: Use a custom component for this, I don't think we should need a library for it -->
```

Comment:
I agree, but I heard that this has better performance somewhere

Reply:
Alright, we'll look into it later